### ISSUE TITLE
misleading libtbx.configure output

### ISSUE THREAD

**Issue by Anthchirp** [conda-forge/cctbx-base-feedstock#3]
When configuring new modules in an otherwise clean conda environment with `cctbx-base` installed, the sources in `/modules/` and running in an empty `/build/`:
```
$ libtbx.configure dxtbx dials xia2
dxtbx is already installed
dials is already installed
xia2 is already installed
(...)
```

**Comment by Anthchirp** [comment]
the configuration is otherwise successful, just the output is misleading

**Comment by bkpoon** [comment]
I'll update the check. It was a simple check to get things working first.

**Comment by bkpoon** [comment]
This should be fixed in `2020.9`. Let me know if you're still seeing it and how you're setting up your environment and directories.

**Comment by Anthchirp** [comment]
Confirmed fixed
